README.md

```diff
@@ -213,3 +213,13 @@ pouch run -it --rm --runtime=rune \
   -e ENCLAVE_RUNTIME_ARGS=occlum_instance \
   occlum-app
 ```
+
+In addition, both `pouch` and `rune` support to configure `annotation` options to run container images, so you also can run pouch, pouchd and rune with Occlum container images refer to
```

Comment:
`pouch` supports to configure annotation` options to run container image. You can run pouch with annotations instead of environment variables.